venafi/resource_venafi_certificate.go

```diff
@@ -296,7 +392,7 @@ func getCertDuration(cert *x509.Certificate) (duration time.Duration) {
 // returns a boolean value of true if expiration_window is greater and logs it in a message, else returns false and doesn't log a message
 func validExpirationWindow(certDuration time.Duration, expirationWindowHours time.Duration) (boolean bool) {
 	if certDuration < expirationWindowHours {
-		log.Printf("certificate validity duration %s is less than configured expiration window %s", certDuration, expirationWindowHours)
+		log.Printf("[INFO] certificate validity duration %s is less than configured expiration window %s", certDuration, expirationWindowHours)
```

Comment:
in this case. can we use tflog

Reply:
Same case as [here](https://github.com/Venafi/terraform-provider-venafi/pull/92#discussion_r868138044)